frontend/src/routes/FacilityBooking/ViewBookingDailyView/index.tsx

```diff
@@ -63,19 +63,19 @@ export default function ViewBookingDailyView() {
     }
   }, [date])
 
+  const bookingFacilityIcon = (
+    <UnorderedListOutlined
+      style={{ color: 'black', fontSize: '150%' }}
+      onClick={() => console.log('link to booking page')}
```

Comment:
What is stopping us from linking it now?

Reply:
I think the revamp booking page not done?